### PR TITLE
Feature/pythrow

### PR DESCRIPTION
**Pull request by nsoblath**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -48,6 +52,18 @@ set_option( Scarab_BUILD_AUTHENTICATION TRUE )
 
 pbuilder_add_submodule( Scarab scarab/library )
 
+if( Dripline_BUILD_PYTHON )
+    message( STATUS "Dripline-cpp is being built with Python" )
+    set( Scarab_BUILD_PYTHON TRUE )
+    add_definitions( -DDL_PYTHON )
+    add_definitions( -DPYTHON_THROW_REPLY_KEYWORD=${Dripline_PYTHON_THROW_REPLY_KEYWORD} )
+    include_directories( ${PYTHON_INCLUDE_DIRS} )
```
Comment (laroque) [inline CMakeLists.txt]: Is this building based on an assumption that pybind11 is included and called findPythonLibs already? Is that safe to assume?

Comment (nsoblath) [reply CMakeLists.txt]: pybind11 is included in scarab.  So I at least know that it's going to be available.  By setting `Scarab_BUILD_PYTHON` to TRUE, we ensure that we're finding the python libraries, etc.

Comment (nsoblath) [reply CMakeLists.txt]: Here's what we do in scarab's CMakeLists.txt file:
```
# Pybind11
if( Scarab_BUILD_PYTHON )
    set( PYBIND11_INSTALL ON CACHE BOOL "Pybind11 option to install itself" FORCE )
    add_subdirectory( external/pybind11 )
    include_directories( external/pybind11/include )    
endif( Scarab_BUILD_PYTHON )
```

Comment (laroque) [reply CMakeLists.txt]: ah okay, so that's good

**Pull request merged**